src/Umbraco.Infrastructure/Media/ImageSharpImageUrlGenerator.cs

```diff
@@ -1,68 +1,88 @@
-﻿using System.Collections.Generic;
+using System;
+using System.Collections.Generic;
 using System.Globalization;
+using System.Linq;
 using System.Text;
+using SixLabors.ImageSharp;
 using Umbraco.Cms.Core.Media;
 using Umbraco.Cms.Core.Models;
-using Umbraco.Extensions;
 
 namespace Umbraco.Cms.Infrastructure.Media
 {
     public class ImageSharpImageUrlGenerator : IImageUrlGenerator
     {
-        public IEnumerable<string> SupportedImageFileTypes => new[] { "jpeg", "jpg", "gif", "bmp", "png" };
+        private static readonly string[] s_supportedImageFileTypes = Configuration.Default.ImageFormats.SelectMany(f => f.FileExtensions).ToArray();
+
+        public IEnumerable<string> SupportedImageFileTypes { get; } = s_supportedImageFileTypes;
 
         public string GetImageUrl(ImageUrlGenerationOptions options)
         {
-            if (options == null) return null;
+            if (options == null)
+            {
+                return null;
+            }
 
-            var imageProcessorUrl = new StringBuilder(options.ImageUrl ?? string.Empty);
+            var imageUrl = new StringBuilder(options.ImageUrl);
 
-            if (options.FocalPoint != null) AppendFocalPoint(imageProcessorUrl, options);
-            else if (options.Crop != null) AppendCrop(imageProcessorUrl, options);
-            else if (options.DefaultCrop) imageProcessorUrl.Append("?anchor=center&mode=crop");
-            else
+            bool queryStringHasStarted = false;
+            void AppendQueryString(string value)
             {
-                imageProcessorUrl.Append("?mode=").Append((options.ImageCropMode ?? ImageCropMode.Crop).ToString().ToLower());
+                imageUrl.Append(queryStringHasStarted ? '&' : '?');
+                queryStringHasStarted = true;
 
-                if (options.ImageCropAnchor != null) imageProcessorUrl.Append("&anchor=").Append(options.ImageCropAnchor.ToString().ToLower());
+                imageUrl.Append(value);
             }
+            void AddQueryString(string key, params IConvertible[] values)
+                => AppendQueryString(key + '=' + string.Join(",", values.Select(x => x.ToString(CultureInfo.InvariantCulture))));
```

Comment:
I've added these local methods to make it easier to add query string parameters, so we don't have to deal with conditionally adding the first `?` or constructive `&` and always format the values using the invariant culture.